static/app/utils/replays/hooks/useReplayList.tsx

```diff
@@ -27,24 +27,15 @@ function useReplayList({eventView, organization}: Options): Result {
     replays: [],
   });
 
-  const init = useCallback(async () => {
-    setData(prev => ({
-      ...prev,
-      isFetching: true,
-    }));
```

Comment:
Doesn't this mean that we will never get a loading state again after the first load?